### PR TITLE
add an option for a subtle border on some UI elements

### LINKED ISSUE #92: Add option to enable borders in "catppuccin" theme
Hi,

I would like to suggest adding an option in the "catppuccin" theme to enable borders on various UI elements, as shown in the following manual example:

```json
"workbench.colorCustomizations": {
    "activityBar.border": "#ffffff15",
    "button.secondaryHoverBackground": "#ffffff15",
    "checkbox.border": "#ffffff15",
    "editorGroupHeader.tabsBorder": "#ffffff15",
    "panel.border": "#ffffff15",
    "panelInput.border": "#ffffff15",
    "sideBar.border": "#ffffff15",
    "sideBarSectionHeader.border": "#ffffff15",
    "statusBar.border": "#ffffff15",
    "tab.border": "#ffffff15",
    "tab.unfocusedActiveBorderTop": "#ffffff15",
    "titleBar.border": "#ffffff15"
}
```
Currently, it's possible to customize the colors of various UI elements in the "settings.json" configuration file using the "workbench.colorCustomizations" property. However, this option does not include the ability to enable or disable borders. The ability to enable borders could be useful for users who want a more defined and contrasted UI.

Therefore, I suggest adding the "catppuccin.activeBorder" option in the "catppuccin" theme to allow users to enable or disable borders on various UI elements.

Thank you for considering this suggestion.

### PR DESCRIPTION
**Pull request by backwardspy**
i had a go at this to see what it would look like.

<details>
  <summary>latte</summary>

latte

</details>
<details>
  <summary>frappé</summary>

frappe

</details>
<details>
  <summary>macchiato</summary>

macchiato

</details>
<details>
  <summary>mocha</summary>

mocha

</details>

let me know your thoughts!

closes #92

### CODE REVIEW

File: src/theme/tokenColors.ts
Comment (andreasgrafen) [inline src/theme/tokenColors.ts:1]: Sneaky spelling fixes; I see! :D

**Review by andreasgrafen — APPROVED** [review]

**Comment by andreasgrafen** [comment]
I'm all for it and thus I've approved the changes. I'll leave the PR open for now in case someone wants to add their opinion. c:

@nekowinston – the latest commits are all yours..
What's your opinion on this?

File: src/theme/tokenColors.ts
Comment (backwardspy) [reply src/theme/tokenColors.ts:1]: i kept them to their own commit at least! 😁

**Review by sgoudham — APPROVED** [review]
LGTM

**Review by nekowinston — APPROVED** [review]

**Pull request merged**